ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/DeferralForm.tsx

```diff
@@ -56,6 +94,10 @@ function DeferralForm({ cves, scopeContext, onCancel }: DeferralFormProps) {
         );
     }
 
+    function setExpiry(expiry: DeferralValues['expiry']) {
+        setValues((prev) => ({ ...prev, expiry })).catch(() => {});
```

Comment:
If the `catch` is to prevent linter error, an alternative I have tried is `return setValues(…)` idiom.

In a few situations, caller handles errors. In the rest, what linter does not know will not hurt it ;)